### ISSUE TITLE
IntegerUpDown must have IsTabStop="True" by default

### ISSUE THREAD

**Issue by xceedsoftware** [xceedsoftware/wpftoolkit#796]
<b>_Dennis_[CodePlex]</b> <br />IntegerUpDown must have IsTabStop=quotTruequot by default.
Moreover, I have no idea, why this property should be quotFalsequot - from the point of keyboard navigation there's no difference for
IntegerUpDown from, say, TextBox.

**Comment by xceedsoftware** [comment]
<b>BoucherS[CodePlex]</b> <br />Correction : Fixed in v2.2.1

**Comment by xceedsoftware** [comment]
<b>BoucherS[CodePlex]</b> <br />Hi, 

This is fixed. It will be included in v2.4
